crates/rover-client/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "rover-client"
+description = "an http client for making graphql requests for the rover CLI"
+version = "0.0.0"
+authors = ["Apollo Developers <opensource@apollographql.com>"]
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+graphql_client = "0.9"
+http = "0.2.1"
```

```suggestion
http = "0.2"
```

this is a personal thing and we might disagree, but i like to leave out the patch release in sub-crates, that way if there is another version of `http` used in `rover`, we can specify the patch release there and we won't bring in the bloat of two different versions of the same code